template/template.go

```diff
@@ -94,46 +94,7 @@ var DefaultFuncs = FuncMap{
 	"toUpper": strings.ToUpper,
 	"toLower": strings.ToLower,
 	"title":   strings.Title,
-	// sortedPairs allows for in-order iteration of key/value pairs.
-	"sortedPairs": func(m map[string]string) []Pair {
-		var (
-			pairs     = make([]Pair, 0, len(m))
-			keys      = make([]string, 0, len(m))
-			sortStart = 0
-		)
-		for k := range m {
-			if k == string(model.AlertNameLabel) {
-				keys = append([]string{k}, keys...)
-				sortStart = 1
-			} else {
-				keys = append(keys, k)
-			}
-		}
-		sort.Strings(keys[sortStart:])
-
-		for _, k := range keys {
-			pairs = append(pairs, Pair{k, m[k]})
-		}
-		return pairs
-	},
-	"firing": func(alerts []Alert) []Alert {
-		res := []Alert{}
-		for _, a := range alerts {
-			if a.Status == string(model.AlertFiring) {
-				res = append(res, a)
-			}
-		}
-		return res
-	},
-	"resolved": func(alerts []Alert) []Alert {
-		res := []Alert{}
-		for _, a := range alerts {
-			if a.Status == string(model.AlertResolved) {
-				res = append(res, a)
-			}
-		}
-		return res
-	},
+	"join":    strings.Join,
```

Comment:
I find the use of `index` very unintuitive and not worth the few saved characters. Especially in this case where we want to access a method.

I changed it already, the `{{ with }}` remains and inside it's ``{{ .Values | join " " }}`.